index.html

```diff
@@ -79,15 +80,21 @@ <h2 class="mb3">Python Panam&aacute; Meetup Vol. 24</h2>
                 <section class="pt2 pb2 align--center">
                     <h3 class="mb1">Expositores</h3>
                     <div class="grid-row">
-                            <div class="grid-column span-2 mt2"></div>
-                            <div class="grid-column span-2 mt2"></div>
-                        <div class="grid-column span-2 mt2">
-                            <img class="mb1" src="img/woman1.jpg" alt="Alyssa Long">
-                            <p class="small">Alyssa Long</p>
+                        <div class="grid-column span-3 mt3">
+                            <img class="mb1" src="img/staff/arnulfo.jpg" alt="Alyssa Long">
+                            <p class="small">MC: Jos&eacute; Arnulfo</p>
                         </div>
-                        <div class="grid-column span-2 mt2">
+                        <div class="grid-column span-3 mt3">
+                            <img class="mb1" src="img/staff/marta.jpg" alt="Alyssa Long">
+                            <p class="small">Marta Maleyka</p>
+                        </div>
+                        <div class="grid-column span-3 mt3">
+                            <img class="mb1" src="img/staff/adriaan.jpg" alt="Jonathan Foster">
+                            <p class="small">Adriaan Beiretz</p>
+                        </div>
+                        <div class="grid-column span-3 mt3">
                             <img class="mb1" src="img/man1.jpg" alt="Jonathan Foster">
```

Comment:
Favor insertar la foto que aparece en su perfil [aqui](https://www.linkedin.com/in/santosrivero/).

Reply:
Listo, va en el próximo commit